### PR TITLE
fix: reduced space below speaker

### LINKED ISSUE #5917: Mobile schedule: Fix details and standardize "Join Video" button
Please enhance the following:
- [ ] Reduce space below speaker
- [x] Standardize "Join Video" button, keep icon on the left (experiment to make it look good and display it in the same way in all sessions)
- [x] Always align "Join Video" button to the top right in all sessions (as you can see in below screenshot it moved down a bit)

Screenshot_20201204_230621_com android chrome

### PR DESCRIPTION
**Pull request by ranjitkshah**
Fixes #5917 

#### Short description of what this resolves:
self-explanatory

#### Screenshot
before:
image

after:
image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by ranjitkshah** [comment]
scss classes are overrided (or not working) due to ` ui grid wide column` that's why using instyling

**Review by MeghalBisht — DISMISSED** [review]

File: app/components/public/session-item.hbs [hunk 1]
```diff
@@ -48,7 +48,7 @@
           {{/if}}
         </div>
       {{else}}
-        <div class="left floated eleven wide column">
+        <div class="left floated eleven wide column" style="padding-bottom: 0 !important;" >
           <div class="d-flex wrap">
             {{#each @session.speakers as |speaker|}}
               <div class="d-flex items-center speaker-card pr-2 pb-4">
```
Comment (afzal442) [inline app/components/public/session-item.hbs:54]: plz make pb-0 and define it.

File: app/components/public/session-item.hbs [hunk 2]
```diff
@@ -48,7 +48,7 @@
           {{/if}}
         </div>
       {{else}}
-        <div class="left floated eleven wide column">
+        <div class="left floated eleven wide column" style="padding-bottom: 0 !important;" >
```
Comment (iamareebjamal) [inline app/components/public/session-item.hbs:51]: Use class

File: app/components/public/session-item.hbs [hunk 3]
```diff
@@ -61,7 +61,7 @@
             {{/each}}
           </div>
         </div>
-        <div class="right floated five wide column">
+        <div class="right floated five wide column" style="padding-top: 0.25rem !important" >
```
Comment (iamareebjamal) [inline app/components/public/session-item.hbs:64]: class

File: app/components/public/session-item.hbs (hunk 2 above)
Comment (ranjitkshah) [reply app/components/public/session-item.hbs:51]: > scss classes are overrided (or not working) due to ` ui grid wide column` that's why using instyling

I tried in utils.scss but it's not working @iamareebjamal

Comment (iamareebjamal) [reply app/components/public/session-item.hbs:51]: It'll work

File: app/components/public/session-item.hbs (hunk 1 above)
Comment (ranjitkshah) [reply app/components/public/session-item.hbs:54]: no, it will become clumsy when there are multiple speakers

**Comment by ranjitkshah** [comment]
image
pls check @iamareebjamal

**Comment by iamareebjamal** [comment]
Yeah, stupid semantic classes use !important themselves. Use inline style for pb-0

File: app/components/public/session-item.hbs [hunk 4]
```diff
@@ -61,7 +61,7 @@
             {{/each}}
           </div>
         </div>
-        <div class="right floated five wide column pt-1" >
+        <div class="right floated five wide column" style="padding-top: .25rem !important;" >
```
Comment (iamareebjamal) [inline app/components/public/session-item.hbs:64]: Not here

Comment (ranjitkshah) [reply app/components/public/session-item.hbs:64]: classes r not working here also

Comment (iamareebjamal) [reply app/components/public/session-item.hbs:64]: Let me think of it

Comment (iamareebjamal) [reply app/components/public/session-item.hbs:64]: Try mt class

Comment (ranjitkshah) [reply app/components/public/session-item.hbs:64]: mt classes (none of utlis classes) are not working there , and there is extra space due to padding

**Pull request merged**